savant/app/query.py

```diff
@@ -153,3 +153,17 @@ def new_users(self):
         self.conn.commit()
         
         return self.new_users
+
+    def null_cases(self):
+        self.cur.execute("select store_debs(%s, %s ,%s, %s)", ('stat', None, 'test', 'test'))
+        self.conn.commit()
+        self.cur.execute("select store_debs(%s, %s ,%s, %s)", ('stat', None, 'test', 'test'))
+        self.conn.commit()
+        self.cur.execute("select count(*) from debs where stat=%s and name is null and version=%s and architecture=%s", ('stat', 'test', 'test'))
+        count = self.cur.fetchone()
+        # self.assertEqual(count[0], 1)
```

Comment:
This solution is more readable and understandable than `if count...`. 

If possible, please un-comment this and remove lines 166-169.

If that is not possible, please remove this commented line of code.

Reply:
assert only works in unit testing I guess

Reply:
I noticed this is `self.assert`. Does plain old `assert` work?